Automattic/sensei
# Review learning mode default spacings

Fix #6785 
Fix [6802](https://github.com/Automattic/sensei/issues/6802)

## Proposed Changes
*  Update the learning mode spacings
*  Update the menu animation - it is still not the final version but we can use it as a reference to check the spacings.
*  Update blocks to render the same HTML that the frontend is rendering. 
*  Fix lesson count spacing issue.
*  Add more classes to the default template block.
*  Rename some classes to use the names generated by Gutenberg. 
* Remove extraneous layout.scss 


## Extra Notes
1. I updated the mobile menu animation because any other solution that I found will just require rework quickly, the animation card is still required to adjust the final details.

2. There are a lot of opportunities to improve the CSS files. E.g. remove the mobile.scss,  reduce the number of variables spread on the scss files, or rethink the scss file structure). It can be addressed in future PRs. Feel free to leave notes here about it, but they should not block the merge. 

3. I found spacing differences when the user is able to see the admin-bar( admin user/teacher user) and when the user is a regular student.


## Testing Instructions
1. `npm run start` 
2.  Install and active Astra
4.  Enable the learning mode
5. Reset any customization applied to the Lesson Default template. 
6. Use a student user to go to a to a lesson 
7. Compare the spacings with the Figma layout (Please ignore (Exit Course + Course Progress) positions on mobile when the menu is opened)
8. Use a teacher/admin and check again if all is fine.


## Pre-Merge Checklist

- [x] New UIs match the designs - Except the elements that I described above.
